deputy/Makefile

```diff
@@ -1,5 +1,5 @@
 
-DEPUTYDISTDIR = ""
```

Comment:
We should also place a Makefile at the top of mlpiper that will compile all the python modules ... this way the pom of reflex-altos and reflex-common will not point outside the dir.

Reply:
Yes, I know.
Let's do it as the next step, in a different patch?!

Reply:
ok